app/assets/stylesheets/application.scss

```diff
@@ -28,11 +29,6 @@
   }
 }
 
-a:hover {
```

Comment:
Removing this because it looks terrible when the background colour is something other than white, but happy to be convinced otherwise